MANUAL.md

```diff
@@ -251,6 +253,15 @@ The following parameters are defined:
     when provisioning the persistent storage for the database.
     Defaults to `postgres`.
 
+  - `postgres_max_connections` - How many connections the internal
+    Postgres DB should maintain at once. Only used if internal
+    DB is deployed. Defaults to `100`.
+
+  - `postgres_vip` - What VRRP VIP to use for the HAProxy/keepalived.
+
+If the `local-ha-db` feature is chosen, a VRRP VIP must be manually
+provided either via the Wizard or as `postgres_vip`
+
 ### Using an External MySQL / MariaDB Database
```

Comment:
Treat `local-ha-db` as a separate flag, either duplicating the docs for `local-db`, or by just introducing another section and leading with "all the things you can do with `local-db` (see previous section), plus these:"